src/Microsoft.DotNet.Analyzers.Compatibility.Tests/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+{
+    "shadowCopy": false
```

Comment:
Can you explain why this is needed?

Reply:
I was getting the same error as https://github.com/dotnet/roslyn/issues/6052 so I wanted the noshadow option was a workaround.